stack_rnn_LM.py

```diff
@@ -91,6 +92,18 @@ def forward(self, sentence, label=None):
 
         stacked_h = torch.stack(h_all_words, dim=1)
         logits = self._classifier(stacked_h)
+
+        # TODO: there must be a better way to do this?
+        final_logits_mask = np.zeros((mask.shape[0],))

```

Comment:
This block is to deal with the mask; replacing this entire chunk with `final_logits = logits[:,-1,:]` (which will incorrectly include padding sometimes) barely effects performance (53 -> 55)